clients/web/src/lib/api/mutations/CancelRestriction.mutation.ts

```diff
@@ -0,0 +1,31 @@
+import {gql} from "@urql/core";
+import {client} from "../client";
+
+interface CancelRestrictionResponse {
+    id: string;
+}
+
+interface CancelRestrictionVariables {
+    id: number;
+    expiration: Date;
+}
+
+const mutationString = gql`
+    mutation (
+        $id: Int!
+        $expiration: DateTime!
+    ){
+        manuallyExpireMemberRestriction(id: $id, manualExpiration: $expiration, manualExpirationReason:"From admin interface") {
+            memberId
+            id
+        }
+    }
+`;
+
+export const cancelRestrictionMutation = async (vars: CancelRestrictionVariables): Promise<CancelRestrictionResponse> => {
```

Comment:
Can we call this expireRestriction or something? Also, this should accept text from an input from the user unbanning them why they are.

Reply:
done